src/index.html

```diff
@@ -12,7 +12,54 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
There should be an empty line between sibling blocks of HTML, but not between parent and child elements. Please add an empty line after the closing </div> of each '.stars' block.